athena-scripts/redshift_scripts/sp_setup_conformed_schema_statements.sql

```diff
@@ -140,7 +140,8 @@ INSERT INTO conformed.REF_RELYING_PARTIES(CLIENT_ID,CLIENT_NAME,DISPLAY_NAME)
 ('DVUDWXsy0io7wDBH5LA5IEkEH5U',	'Plan Your Future','PYF - Plan Your Future'),
 ('9uEx86ZHEp8ycgdHNqC8VK87E1A',	'Find and Use an API','DFE - Find and Use an API'),
 ('tPCCSyoMaFNbLTt0gEW609h15Uc',	'Fisheries Permit','MF - Fisheries Permit'),
-('Gk-D7WMvytB44Nze7oEC5KcThQZ4yl7sAA',	'Immigration Advisor Self-Service','OISC - Immigration Advisor Self-Service');
+('Gk-D7WMvytB44Nze7oEC5KcThQZ4yl7sAA',	'Immigration Advisor Self-Service','HO - Immigration Advice Authority'),
+('XbPzF-ccO0utCxlifxSyA4Ng0API2XTCQQ',	'Visit someone in prison','MOJ - Visit someone in prison');
```

Comment:
Is this part of https://govukverify.atlassian.net/browse/DAC-3522 or is it a separate piece of work?